### ISSUE TITLE
Feature request: center window if in floating mode 

### ISSUE THREAD

**Issue by wanbinkimoon** [nikitabobko/AeroSpace#494]
**Feature request**

## Checklist
- [x] I've searched for duplicates. My search queries were (please don't forget to search in closed issues too):
  - "center window"
  - "move floating windows"

- [x] I've read the documentation https://nikitabobko.github.io/AeroSpace/guide & https://nikitabobko.github.io/AeroSpace/commands

- [X] I've searched in the documentation. My search queries were:
  - "center window"

```
aerospace CLI client version: 0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201
AeroSpace.app server version: 0.14.2-Beta 0cb8dbdfc5ee73b8cbc200f175f467ebead55201
```

I come from a much simpler window manager and I have been using aerospace for few weeks, I love the software but something that I really miss is a command to center a window in the screen when the window is in floating mode. 

I'm not sure of the implementation cost but I think it could be a quick win for everyone.

**Comment by FrancoLab** [comment]
This would be excellent, especially with the capability to resize floating windows.

**Comment by Alberto-B-Lopez** [comment]
Yes please add this i really miss the feature from yabi where i can just detach to a floating window and it auto centers.

**Comment by tim-hilde** [comment]
Any news on this?

**Comment by AmmarCodes** [comment]
If someone is using Raycast they can use the Window Management > Center in the meanwhile.

**Comment by mohammadazeemwani** [comment]
> If someone is using Raycast they can use the Window Management > Center in the meanwhile.

Where? I am unable to find it ?
